packages/suite-desktop/src/support/screens/TorLoadingScreen.tsx

```diff
@@ -177,7 +177,7 @@ export const TorLoadingScreen = ({ callback }: TorLoadingScreenProps) => {
 
     return (
         <ThemeProvider>
-            <Wrapper>
+            <Wrapper data-test="@tor-loading-screen">
```

Comment:
I slightly changed the component structure in https://github.com/trezor/trezor-suite/pull/6084, so perhaps you'd have to rebase. The changes will be minimal though ;)

Reply:
Thanks for mentioning it, I updated it to the new code.